internal/worker/clienterrors/errors.go

```diff
@@ -27,6 +27,8 @@ const (
 	ErrorRPMMDError        ClientErrorCode = 23
 	ErrorEmptyPackageSpecs ClientErrorCode = 24
 	ErrorDNFRepoError      ClientErrorCode = 25
+
+	ErrorJobDependency ClientErrorCode = 26
```

Comment:
Wrong formatting

Reply:
Well, kind of.

I found the formatting weird to start with. The error codes seemed to be assigned without leaving any room for new codes, while being split into two blocks. The first block seems to be related to job errors in general, while the second is related to depsolving and DNF. The new error fits more to the first block, but it felt weird to add it to it while breaking the actual error code ordering.

I'll probably squash all errors into one block without any newline separating them into multiple blocks.

Reply:
One block would be fine. The actual value of the codes doesn't matter as long as they're not changed.

Reply:
Ah, I missed the new newline between 25 and 26. Nevemind, do whatever feels most reasonable to you. ;)